python/paddle/v2/trainer.py

```diff
@@ -51,7 +51,8 @@ def __init__(self,
                  update_equation,
                  extra_layers=None,
                  is_local=True,
-                 pserver_spec=None):
+                 pserver_spec=None,
+                 use_etcd=True):
```

Comment:
Maybe we can make the default requires less dependency, by defaulting `use_etcd` to False.